index.html

```diff
@@ -4292,6 +4306,489 @@ <h3>Proof Formats</h3>
       </section>
     </section>
 
+    <section class="normative">
+      <h2>Algorithms</h2>
+
+      <p>
+This section contains algorithms that can be used by implementations to
+perform common operations.
+      </p>
+
+      <p class="note" title="Implementers can include additional checks, warnings, and errors.">
+Implementers are advised that the algorithms in this section contain the bare
+minimum set of checks used by implementations to test conformance to this
+specification. Implementations are expected to provide additional checks that
+report helpful warnings for developers to help debug potential issues.
+Similarly, implementations are likely to provide additional checks that
+could result in new types of errors being reported in order to stop harmful
+content. Any of these additional checks might be integrated into future
+versions of this specification.
+      </p>
+
+      <section class="normative">
+        <h3>Verification</h3>
+
+        <p>
+This section contains an algorithm that <a>conforming verifier
+implementations</a> MUST run when verifying a <a>verifiable credential</a> or a
+<a>verifiable presentation</a>. This algorithm takes a byte sequence (bytes
+<var>inputBytes</var>) and a media type (string <var>inputMediaType</var>) as
+inputs, and produces an object (<var>result</var>) that contains the
+following:
+        </p>
+
+        <ul>
+          <li>
+a status (boolean <var>status</var>)
+          </li>
+          <li>
+a conforming document (object <var>document</var>)
```

Comment:
```suggestion
a <a>conforming document</a> (object <var>document</var>)
```

Reply:
@msporny making this a reference to the conformance section causes all the MUST statements to be clearly required... later if @jyasskin still wants each property repeated, and validated, we can still do that, but I would expect to see date format checks, url format checks, duplicate json key checks, etc... otherwise we are still repeating the normative requirements with only partial coverage in the "algorithm".